Services/Database/interfaces/interface.ilTableLockInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Class ilTableLockInterface
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilTableLockInterface {
+
+	/**
+	 * @param bool $lock_bool
+	 * @return ilTableLockInterface
+	 */
+	public function lockSequence($lock_bool);
+
+
+	/**
```

Comment:
I would strongly prefer to come to a version workiung for all requirements, if this state is reached, we should focus on missing examples, semantics, explanations, whatever.

The point is, i cannot finalize a contract although i have less time until the alpha freeze.

I also think, Fabian want to do this job once, when we should need a further fix this work is t be done again or at least to be revalidated.

Best Björn